tests/docs/smoke-all/2023/09/11/6800.qmd

```diff
@@ -0,0 +1,22 @@
+---
+title: "test"
+format:
+    revealjs:
+        footer: "hello world"
+keep-md: true
+_quarto:
+  tests:
+    revealjs:
+      ensureHtmlElements:
+        - ["div.footer"]
```

Comment:
I believe we should test `.footer-default` class, as this will be the class used for a footer set globally in `format` YAML key. (what our support plugin target). or test both classes maybe. 

Test is ok as of now, so probably too picky but just trying to be precise to cover more ground.